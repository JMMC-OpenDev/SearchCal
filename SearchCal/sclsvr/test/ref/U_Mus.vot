version https://git-lfs.github.com/spec/v1
oid sha256:fd28748646b0782b17a8ba7a69aacd7af801932e3a8571abb54c9abcae0c8436
size 159430
