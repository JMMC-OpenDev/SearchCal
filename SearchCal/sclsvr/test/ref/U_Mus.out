version https://git-lfs.github.com/spec/v1
oid sha256:af90231b608c71b4bac314a58093d4a37a4dcd8e05715ee862571779cc549fb3
size 201740
